### ISSUE TITLE
Cd Key In Use by Me?

### ISSUE THREAD

**Issue by KryptoD2** [kolton/d2bot-with-kolbot#863]
My CD Keys have been working fine but every once in a while I'll get "CD Key is in use by xxxx"... These are brand new keys I brought from the battlenet website... how is that even possible???

**Comment by cameronpfs** [comment]
It is a standard bug in d2.

**Comment by aim2kill** [comment]
Yeah, happen to me to in 3 cd key, make sure u close all game instances before closing the manager. Aleast this help me avoid it a bit
